### PR TITLE
chore: clean unused code and add `singleton` config to metrics task

### PR DESCRIPTION
**Pull request by silenceqi**
## What does this PR do
clean unused code 
add `singleton` config to metrics task

## Standards checklist

- [ ] The PR title is descriptive
- [ ] The commit messages are [semantic](https://www.conventionalcommits.org/)
- [ ] Necessary tests are added
- [ ] Updated the release notes
- [ ] Necessary documents have been added if this is a new feature
- [ ] Performance tests checked, no obvious performance degradation

### CODE REVIEW

**Review by medcl — APPROVED** [review]

**Review by medcl — CHANGES_REQUESTED** [review]
Make sure you check and passed the `Standards checklist`

**Review by luohoufu — APPROVED** [review]

**Comment by silenceqi** [comment]
> Make sure you check and passed the `Standards checklist`

Updated release notes

File: docs/content.en/docs/release-notes/_index.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ Information about release notes of INFINI Framework is provided here.
 
 ### Improvements
```
Comment (medcl) [inline docs/content.en/docs/release-notes/_index.md:17]: your release notes should place under `Latest (In development)`

**Review by medcl — CHANGES_REQUESTED** [review]

**Review by medcl — APPROVED** [review]

File: docs/content.en/docs/release-notes/_index.md [hunk 2]
```diff
@@ -8,10 +8,14 @@ title: "Release Notes"
 Information about release notes of INFINI Framework is provided here.
 
 ## Latest (In development)
+### Features
```
Comment (medcl) [inline docs/content.en/docs/release-notes/_index.md:11]: Your release notes is quite confusing and i don't really get what you mean, can you rephrase the wording, can be easily fixed by using ChatGPT.

**Review by medcl — CHANGES_REQUESTED** [review]

File: docs/content.en/docs/release-notes/_index.md [hunk 3]
```diff
@@ -8,10 +8,14 @@ title: "Release Notes"
...
+- Set the metric collection task to singleton mode (#17)
 
 ### Breaking changes
 
 ### Bug fix
```
Comment (medcl) [inline docs/content.en/docs/release-notes/_index.md:16]: why not resolve the rest of them?

**Review by medcl — APPROVED** [review]
